x11-misc/dex/dex-0.10.1.ebuild

```diff
@@ -9,14 +9,13 @@ inherit python-r1
 
 DESCRIPTION="DesktopEntry eXecution - tool to manage and launch autostart entries"
 HOMEPAGE="http://e-jc.de/"
-SRC_URI="https://github.com/jceb/dex/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/jceb/dex/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="amd64 x86 ~amd64-linux ~x86-linux"
```

Comment:
Always drop keywords back to testing when adding a new version.
**ekeyword ~all dex-0.10.1.ebuild**